### DISCUSSION TITLE
Give the possibility to add a custom control popup directly in the xml of a page

### DISCUSSION THREAD

**Discussion by acaliaro** [CommunityToolkit/Maui#837] [General]
Hello. It seems that currently the popup can only be displayed by a c# call ( ShowPopup method). It might be useful to have the possibility to add a popup directly in the xml of a ContentPage, as if it were any CustomView, and display it using an "IsVisible" boolean property

**Comment by pictos** [comment]
I'm not sure if I followed your use case. Can you expand a little bit?

In general, we don't tweak `IsVisible`, to show the popup, we create the view on the fly and _navigate_ to it. I believe you can something fancy to allow that to be used in xaml, like prism does with navigation stuff

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
